src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -117,6 +117,13 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
     _createPropertyAccessors (window, document) {
         var storedDomain = '';
 
+        function hasElementProperty (el, prop) {
```

Comment:
`innerHTML` and etc. are text properties.
Let's do more specified check for this.
```js
function elementHasTextPropertyInProtoChain (el, propName) {
    return typeof el[propName]  === string && !el.hasOwnProperty(propName);
}
```
So, name describes the function behavior and you can remove comment.